modules/api/pkg/provider/kubernetes/constraint_test.go

```diff
@@ -62,8 +62,6 @@ func TestListConstraints(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			t.Parallel()
```

Comment:
I decided to remove this instead of adding linter exception as those tests have only one scenario each.